### DISCUSSION TITLE
Discount Factor

### DISCUSSION THREAD

**Discussion by 101AlexMartin** [pyg-team/pytorch_geometric#9111] [Q&A]
Hi,

I was wondering if this repository has the functionality of the discount factor (usually expressed as \gamma) implemented. The discount factor aims to take into account the loss after each layer of the graph network in the update of the weights.

**Comment by rusty1s** [comment]
There is no such functionality in PyG. Do you mind sharing a paper reference?

**Reply by 101AlexMartin** [reply]
In page 8, you find the role of the discount factor: https://www.sciencedirect.com/science/article/pii/S0378779620303515
